client/my-sites/themes/theme-showcase.jsx

```diff
@@ -305,6 +314,16 @@ class ThemeShowcase extends React.Component {
 							{ translate( 'Install theme' ) }
 						</Button>
 					) }
+					{ this.showPreviousThemeButton() && (
+						<Button
+							className="themes__previous-theme-button"
+							compact
+							href={ `/theme/${ previousTheme }/${ siteSlug }` }
+						>
+							<Gridicon icon="history" />
+							{ translate( 'Previous theme' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **120** times:
`translate( 'Show previous theme' )` *ES Score: 6*

Reply:
I think this works, so I've updated that to this! It also matches the phrase of "Install theme" where a verb is also present.